sdk/cosmos/azure-cosmos/azure/cosmos/database.py

```diff
@@ -222,6 +222,62 @@ def create_container(
 
         return ContainerProxy(self.client_connection, self.database_link, data["id"], properties=data)
 
+    @distributed_trace
+    def create_container_if_not_exists(
+        self,
+        id,  # type: str  # pylint: disable=redefined-builtin
+        partition_key,  # type: Any
+        indexing_policy=None,  # type: Optional[Dict[str, Any]]
+        default_ttl=None,  # type: Optional[int]
+        populate_query_metrics=None,  # type: Optional[bool]
+        offer_throughput=None,  # type: Optional[int]
+        unique_key_policy=None,  # type: Optional[Dict[str, Any]]
+        conflict_resolution_policy=None,  # type: Optional[Dict[str, Any]]
+        **kwargs  # type: Any
+    ):
+        # type: (...) -> ContainerProxy
+        """
+        Create the container if it does not exist already.
+
+        If the container already exists, it is returned.
```

Comment:
Suggestion: `the existing settings are returned. Note: it does not check or update the existing container settings if they differ from what was passed into the method.`

Reply:
Same for database (but with database instead of container)